lib/Widgets/OutgoingTransmissionListEntry.vala

```diff
@@ -37,9 +37,11 @@ public class Drop.Widgets.OutgoingTransmissionListEntry : TransmissionListEntry
      * @param transmission Transmission to build the entry on.
      */
     public OutgoingTransmissionListEntry (OutgoingTransmission transmission) {
-        Object (transmission: transmission);
+        base ();
+        this.transmission = transmission;
```

Comment:
Please add a blank line to keep it consistent with IncomingTrnasmissionListEntry.vala